tmt-app/src/Components/MajorCareerList.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { FlexContainer } from "../Styling/CustomStyling.js";
+import { Container, Typography } from "@mui/material";
+import { CAREERS_LIST, MAJORS_LIST } from "../constants/careerMajorList";
+
+function MajorCareerList() {
+  return (
+    <FlexContainer
+      sx={{
+        alignItems: "flex-start",
+      }}
+    >
+      <Container>
+        <Typography variant="CustomHeading3" sx={{ marginBottom: "1rem" }}>
+          Major
+        </Typography>
+        {MAJORS_LIST.map((item) => (
+          <div key={item.id}>{item.name}</div>
```

Comment:
nit: I generally like wrapping mappings like this in a div to differentiate this block from the adjacent blocks like typography.

Reply:
Side note we could use `React.Fragment` or `<> </>` if we need to wrap w/o creating too many empty divs 🙆🏻‍♂️